dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/AndroidTracer.kt

```diff
@@ -137,5 +139,29 @@ class AndroidTracer internal constructor(
         internal const val TRACE_LOGGER_NAME = "trace"
 
         internal const val TRACE_ID_BIT_SIZE = 63
+
+        /**
+         * Helper method to attach a Throwable to a specific Span.
+         * The Throwable information (Class name, message and Stacktrace) will be added to the
+         * provided Span as special Tags.
+         * @param forSpan as the active Span
+         * @param throwable as the Throwable you wan to log
+         */
+        @JvmStatic
+        fun logThrowable(forSpan: Span, throwable: Throwable) {
```

Comment:
this is not ObjectiveC ;) 
can you rename `forSpan` into `span` please ?

Reply:
🙄 💔